base/uk.ac.stfc.isis.ibex.opis/resources/CCD100/updateControlsForNumberOfChannels.py

```diff
@@ -0,0 +1,33 @@
+from org.csstudio.opibuilder.scriptUtil import PVUtil
+from org.csstudio.opibuilder.scriptUtil import ConsoleUtil
+from time import sleep
+
+def is_active_macro(value):
+    if value is None or len(value)==0:
+        return False  # Empty macro
+    elif len(value)>=3 and value[:2]=="$(" and value[-1]==")":
+        return False  # Unexpanded macro
+    else:
+        return True
+    
+def get_count():
+    count = 0
+    macros = display.getPropertyValue("macros").getMacrosMap()
+    for i in range(1,5):
+        if is_active_macro(macros.get("CCD_"+str(i))):
```

Comment:
Use `.format()` as per standards discussed at code chat

Reply:
```
2018-01-02 16:36:20 ERROR: Error in updateControlsForNumberOfChannels.py on CCD100.
Traceback (most recent call last):
  File "<script>", line 34, in <module>
  File "<script>", line 17, in main
AttributeError: 'str' object has no attribute 'format'
```

As expected, can't be used in CSS